### PR TITLE
stable-1.6.0 merge and release notes

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

Release

**What this PR does/ why we need it**:

1.6.0 release

**Which issue(s) this PR fixes**:

no issue

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [x] The commit message explains the changes and why are needed
* [x] The documentation is updated where needed.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # Release Notes
 
-## stable-1.15-1.5.0, stable-1.16-1.5.0
+## stable-1.15-1.6.0, stable-1.16-1.6.0
+
+* \[cert-manager\] `usages` is no longer definable as part of `issuerRef`, instead it is a key on its own ([#196](https://github.com/mesosphere/kubernetes-base-addons/pull/196), [@sebbrandt87](https://github.com/sebbrandt87))
```
Comment (dkoshkin) [inline RELEASE_NOTES.md:5]: Is this change the reason for the minor bump instead of a patch?

Comment (joejulian) [reply RELEASE_NOTES.md:5]: Each bi-monthly release is a minor bump. Patch releases are done for patches out of the release cadence, to patch a bug. We did that for 1.4 where it broke the velero backup.

Comment (samvantran) [reply RELEASE_NOTES.md:5]: Hm there should be a mention of the easy token refresh in these notes https://jira.d2iq.com/browse/D2IQ-65269?src=confmacro

Comment (joejulian) [reply RELEASE_NOTES.md:5]: Not according to the PR: https://github.com/mesosphere/kubernetes-base-addons/pull/193

Comment (samvantran) [reply RELEASE_NOTES.md:5]: Ah probably oversight. @jr0d this is a new feature worth calling out, yes? This is for the kubectl plugin with dka instructions page

Comment (samvantran) [reply RELEASE_NOTES.md:5]: also dka, dex, and opsportal addons all got updated ... maybe we should include something in the release notes about those (unless all related to security plugin)

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
 # Release Notes
 
-## stable-1.15-1.5.0, stable-1.16-1.5.0
+## stable-1.15-1.6.0, stable-1.16-1.6.0
+
+* \[dex-k8s-authenticator\]: Now supports a kubectl credentials plugin for automatically managing identity tokens. Instructions for downloading the plugin and configuring kubectl can be found at `https://\u003ccluster-ip\u003e/token/plugin`. ([#212](https://github.com/mesosphere/kubernetes-base-addons/pull/212), [@jr0d](https://github.com/jr0d))
```
Comment (samvantran) [inline RELEASE_NOTES.md:5]: accidental encoding in the link?` \u003e`

Comment (joejulian) [reply RELEASE_NOTES.md:5]:
```suggestion
* \[dex-k8s-authenticator\]: Now supports a kubectl credentials plugin for automatically managing identity tokens. Instructions for downloading the plugin and configuring kubectl can be found at `https://<cluster-ip>/token/plugin`. ([#212](https://github.com/mesosphere/kubernetes-base-addons/pull/212), [@jr0d](https://github.com/jr0d))
```

**Comment by joejulian** [comment]
Admin merging with unsigned dependabot commit.

**Pull request merged**